templates/corporate/hello.html

```diff
@@ -1,562 +1,402 @@
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-3x.webp') }} 3x"
+                      media="(min-width: 941px)"/>
+                    <source class='appshot-1__img'
+                      type="image/webp"
+                      srcset="{{ static('images/landing-page/hello/screen-1-mobile-1x.webp') }},
+                      {{ static('images/landing-page/hello/screen-1-mobile-2x.webp') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-mobile-3x.webp') }} 3x"
+                      media="(max-width: 940px)"/>
+                    <source class='appshot-1__img'
+                      srcset="{{ static('images/landing-page/hello/screen-1-mobile-1x.jpg') }},
+                      {{ static('images/landing-page/hello/screen-1-mobile-2x.jpg') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-mobile-3x.jpg') }} 3x"
+                      media="(max-width: 940px)"/>
+                    <img alt="" class='appshot-1__img' src="{{ static('images/landing-page/hello/screen-1-1x.jpg') }}"
+                      srcset="{{ static('images/landing-page/hello/screen-1-1x.jpg') }},
+                      {{ static('images/landing-page/hello/screen-1-2x.jpg') }} 2x,
+                      {{ static('images/landing-page/hello/screen-1-3x.jpg') }} 3x"/>
+                </picture>
+                <div class="cta-buttons">
+                    <a href="/try-zulip">
+                        <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M12 14.9L13.225 12.225L15.9 11L13.225 9.775L12 7.1L10.775 9.775L8.1 11L10.775 12.225L12 14.9ZM3.5 18.5C3.1 18.5 2.75 18.35 2.45 18.05C2.15 17.75 2 17.4 2 17V5C2 4.6 2.15 4.25 2.45 3.95C2.75 3.65 3.1 3.5 3.5 3.5H20.5C20.9 3.5 21.25 3.65 21.55 3.95C21.85 4.25 22 4.6 22 5V17C22 17.4 21.85 17.75 21.55 18.05C21.25 18.35 20.9 18.5 20.5 18.5H3.5ZM1.75 21.5C1.53333 21.5 1.35417 21.4292 1.2125 21.2875C1.07083 21.1458 1 20.9667 1 20.75C1 20.5333 1.07083 20.3542 1.2125 20.2125C1.35417 20.0708 1.53333 20 1.75 20H22.25C22.4667 20 22.6458 20.0708 22.7875 20.2125C22.9292 20.3542 23 20.5333 23 20.75C23 20.9667 22.9292 21.1458 22.7875 21.2875C22.6458 21.4292 22.4667 21.5 22.25 21.5H1.75Z" fill="white" />
+                        </svg>
+                        Try Zulip now
                     </a>
-                </div>
-                <div class="group">
-                    <h2>Mobile</h2>
-                    <a href="{{ apps_page_url }}ios">
-                        <i class="fa fa-mobile-phone platform-icon" aria-hidden="true"></i>
+                    <a href="https://zulip.com/communities/">
```

Comment:
Since this landing page will only be shown on zulip.com, links like this should probably all be relative.

Reply:
Done.